app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <div class="checkbox">
+        <%= form.label :duplicable do %>
+          <%= form.check_box :duplicable %>
+          This stage can serve as a production template.
```

Comment:
not sure why we need this ... if a stage goes to production it can serve as a production template ?
also I expect people will clone a stage and just leave this on, breaking it ...
... if we still want it, then pick a better column name or description ? `Can be used as template for copying / has not unique customizations`